server.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Please you have disabled `eslint` in quite a  number of places, the essence of `linter` is to ensure that you stick to best practices. Instead of disabling it, please kindly fix the issues in your code base.